README.md

```diff
@@ -0,0 +1,53 @@
+# Vouchers4Veggies
+This project is designed and developed by a team of UC Berkeley students through one of [Cal Blueprint](https://calblueprint.org/)'s project teams during the 2022-23 academic year. 
+
+Learn more about [Vouchers 4 Veggies](https://eatsfvoucher.org/) and [Cal Blueprint](https://calblueprint.org/).
+
+---
+## Getting Started
+
+### Prerequisites
+
+Check your installation of `npm` and `node`:
+
+```sh
+node -v
+npm -v
+```
+
+We strongly recommend using a Node version manager like [nvm](https://github.com/nvm-sh/nvm) (for Mac) or [nvm-windows](https://github.com/coreybutler/nvm-windows) (for Windows) to install Node.js and npm. See [Downloading and installing Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+
+### Installation
+
+1. Clone the repo & install dependencies
+   1. Clone this repo
+        ```sh
+        git clone git@github.com:calblueprint/vouchers4veggies.git
```

Comment:
Is there a reason to use SSH rather than HTTPS for cloning the project? I feel like generating SSH keys can get p tedious.

Reply:
Good point -- I think SSH is just known to be more secure and so it has become more of an industry-standard, but you're right in that it might introduce a lot of unnecessary complexity for a project of this scale. I'll update it to highlight both options & link some docs.